rtgui/batchqueuepanel.cc

```diff
@@ -192,7 +192,7 @@ void BatchQueuePanel::init (RTWindow *parent)
     saveFormatPanel->init (options.saveFormatBatch);
 }
 
-// it is expected to have a non null forceOrientation value on Preferences update only. In this case, qsize is ingored and computed automatically
```

Comment:
@heckflosse This is my favorite! 🤣

Reply:
@Floessie I really don't understand :(

Reply:
@heckflosse seems your eyes are ingoring the typo ;-)

Reply:
😂